src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -1,61 +1,69 @@
 import * as React from 'react';
-import { StyleSheet, TouchableNativeFeedback, View } from 'react-native';
+import {StyleSheet, TouchableNativeFeedback, View} from 'react-native';
 
+import FastImage from 'react-native-fast-image';
+import PropTypes from 'prop-types';
 import Image from '../../../components/Image';
 import MemoIcAddCircle from '../../../assets/icons/ic_add_circle';
-import { DEFAULT_PROFILE_PIC_PATH } from '../../../utils/constants';
-import FastImage from 'react-native-fast-image';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
+import {normalize} from '../../../utils/fonts';
+import {CircleGradient} from '../../../components/Karma/CircleGradient';
 
-/**
- * @typedef {Object} ProfilePicturePropsParam
- * @property {function} onImageContainerClick
- * @property {String} profilePicPath
- * @property {Boolean} disabledAddIcon
- */
-/**
- *
- * @param {ProfilePicturePropsParam} props
- */
-const ProfilePicture = ({ onImageContainerClick, profilePicPath, disabledAddIcon = false}) => {
-    const __renderAddIcon = () => {
-        if(!profilePicPath || !disabledAddIcon) return <></>
+const ProfilePicture = ({
+  onImageContainerClick,
+  profilePicPath,
+  disabledAddIcon = false,
+  karmaScore = 0
+}) => {
+  const renderAddIcon = () => {
+    if (!profilePicPath || !disabledAddIcon) return <></>;
 
-        return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />
-    }
-    return <View style={styles.wrapImageProfile}>
-        <TouchableNativeFeedback onPress={onImageContainerClick}>
-            <View style={styles.profileImageContainer}>
+    return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />;
+  };
+  return (
+    <View style={styles.wrapImageProfile}>
+      <TouchableNativeFeedback onPress={onImageContainerClick}>
+        <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(2)}>
+          <View style={styles.profileImageContainer}>
             <Image
-                style={styles.profileImage}
-                source={{
-                uri: profilePicPath
-                    ? `${profilePicPath}`
-                    : DEFAULT_PROFILE_PIC_PATH,
-                }}
-                resizeMode={FastImage.resizeMode.stretch}
+              style={styles.profileImage}
+              source={{
+                uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH
+              }}
+              resizeMode={FastImage.resizeMode.stretch}
             />
-            { __renderAddIcon() }
-            </View>
-        </TouchableNativeFeedback>
+            {renderAddIcon()}
+          </View>
+        </CircleGradient>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `CircleGradient` component is used here to create a circular gradient background for the profile picture based on the user's karma score. This is a good use of the component, but it would be better to handle cases where the `karmaScore` prop is undefined or null. If `karmaScore` is not provided, you could default to a neutral color or another appropriate fallback.



<!-- This is an auto-generated comment by OSS CodeRabbit -->